bibliography.bib

```diff
@@ -0,0 +1,87 @@
+@InProceedings{BOOK1,
+	author=	{Frederic H. Martini, Robert B. Tallitsch, Judi L. Nath},
+	title=	{{Anatomia umana}},
+	booktitle=	{{Anatomia umana }},
+	year=	{2019},
+}
+
+@InProceedings{BOOK2,
+	author=	{Luciano Zocchi},
+	title=	{{Principi di fisiologia}},
+	booktitle=	{{Principi di fisiologia}},
+	year=	{2012},
+}
+
+@misc{LINFOMIAIL,
+	author = {AIL },
+	title = {{Linfomi AIL}},
+	url = {{https://www.ail.it/patologie-e-terapie/patologie-ematologiche/linfomi}},
+}
+
+@misc{img1,
+	author = { },
+	title = {{https://www.pharmacy180.com/article/lymphatic-system-and-immunity-3637/}},
+	url = {{https://www.pharmacy180.com/article/lymphatic-system-and-immunity-3637/}},
+}
+
+@misc{img2,
+	author = { },
+	title = {{https://www.cancer.gov/publications/dictionaries/cancer-terms/def/lymph-system}},
+	url = {{https://www.cancer.gov/publications/dictionaries/cancer-terms/def/lymph-system}},
+}
+
+@misc{LNHAIL,
+	author = {AIL },
+	title = {{LNH AIL}},
+	url = {{https://www.ail.it/patologie-e-terapie/patologie-ematologiche/linfomi/576-linfoma-non-hodgkin}},
+}
+
+@misc{LNHAIMAC,
+	author = {AIMAC },
+	title = {{AIMAC}},
+	url = {{https://www.aimac.it/libretti-tumore/linfomi-non-hodgkin/qual-e-la-differenza-tra-linfoma-di-hodgkin-e-linfomi-non-hodgkin#:~:text=Un%20linfoma%20non%20Hodgkin%20si,dei%20linfomi%20del%20secondo%20tipo}},
+}
+
+@misc{AMERICANCS,
+	author = {The American Cancer Society medical and editorial content team},
+	title = {{Non-Hodgkin Lymphoma Causes, Risk
+Factors, and Prevention}},
+	url = {{https://www.cancer.org/content/dam/CRC/PDF/Public/8718.00.pdf}},
+}
+
+@misc{AIOM,
+	author = {Associazione Italiana di Oncologia Medica},
+	title = {{Linee guida linfomi}},
+	url = {{https://www.aiom.it/wp-content/uploads/2018/11/2018_LG_AIOM_Linfomi.pdf}},
+}
+
+@misc{AIRTUM,
+	author = {Associazione Italiana Registri Tumori},
+	title = {{Cos'è AIRTUM}},
+	url = {{https://www.registri-tumori.it/cms/pagine/cosa-%C3%A9-airtum}},
+}
+
+@misc{AIRC,
+	author = {Associazione Italiana Ricerca Cancro},
+	title = {{Guida ai tumori. Linfoma non-Hodgkin}},
+	url = {{https://www.airc.it/cancro/informazioni-tumori/guida-ai-tumori/linfoma-non-hodgkin}},
+}
+
+@misc{Americanstatistic,
+	author = {American Statistic Center},
+	title = {{Linfoma non-Hodgkin 2022 Estimates}},
+	url = {{https://cancerstatisticscenter.cancer.org/#!/}}
+}
+
+@misc{reteveneta,
+	author = {Rete Ematologica Veneta},
+	title = {{Percorsi Diagnostico Terapeutici Assistenziali (PDTA)
+LINFOMI NON HODGKIN}},
+	url = {{https://rev.aopd.veneto.it/all/PDTA_Linfomi_Non_Hodgkin.pdf}}
+}
+
+@misc{LLS,
+	author = {Leukemia & Lymphoma Society},
```

Comment:
```suggestion
	author = {Leukemia \& Lymphoma Society},
```

try this, [source](https://tex.stackexchange.com/questions/7198/how-to-escape-the-ampersand-character-in-bibtex)